sdk/client/commander_test.go

```diff
@@ -220,13 +222,15 @@ func TestCommander_Recv_Reconnect(t *testing.T) {
 	grpcServer, commandService, dialer := startCommanderMockServer()
 
 	ctx := context.TODO()
-
 	commanderClient := createTestCommanderClient(dialer)
-	commanderClient.WithBackoffSettings(backoff.BackoffSettings{
-		InitialInterval: 100 * time.Millisecond,
-		MaxInterval:     100 * time.Millisecond,
-		MaxElapsedTime:  30 * time.Second,
+	commanderClient.WithProtoBackoffSettings(&proto.Backoff{
```

Comment:
maybe centralize proto.Backoff in the test utils GetMockAgentConfig - you can call GetMockAgentConfig().Server.Backoff then?